2022/25xxx/CVE-2022-25736.json

```diff
@@ -1,18 +1,69 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2022-25736",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
+   "CVE_data_meta": {
+      "ASSIGNER": "product-security@qualcomm.com", 
+      "ID": "CVE-2022-25736", 
+      "STATE": "PUBLIC"
+   }, 
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
```

Comment:
Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```